File: assignment_1/Investor.java
// This class is for people who invest money in the store.
public class Investor {
    int uid;
    int i_id;
    String User_name;
    String contact;
    String email;
    String address;
    String date;
    double amount;

    // Prints all investor details
    void displayInvestor() {
        System.out.println("Investor ID : " + i_id);
        System.out.println("User ID     : " + uid);
        System.out.println("Name        : " + User_name);
        System.out.println("Contact     : " + contact);
        System.out.println("Email       : " + email);
        System.out.println("Address     : " + address);
        System.out.println("Date        : " + date);
        System.out.println("Amount      : Rs." + amount);
    }

    // Shows that investor invested money
    void invest() {
        System.out.println(User_name + " invested Rs." + amount);
    }
}

File: python/investor.py
class Investor:
    def __init__(self):
        self.uid = 0
        self.i_id = 0
        self.User_name = ""
        self.contact = ""
        self.email = ""
        self.address = ""
        self.date = ""
        self.amount = 0.0

    def displayInvestor(self):
        print("Investor ID :", self.i_id)
        print("User ID     :", self.uid)
        print("Name        :", self.User_name)
        print("Contact     :", self.contact)
        print("Email       :", self.email)
        print("Address     :", self.address)
        print("Date        :", self.date)
        print("Amount      : Rs.", self.amount)

    def invest(self):
        print(self.User_name, "invested Rs.", self.amount)